<?xml version="1.0"?>
<launch>


<!-- get the arguments from  -->
<arg name="traj" doc="Configuration profile to use" />
<arg name="debug" default="False" doc="Whether debug is on" />
<arg name="speed_factor" default="1.0" doc="the speed to run the trajectory at" />
<arg name="reps" default="1" doc="The number of reps" />
<arg name="replan" default="false" doc="The number of reps" />

<arg name="arm" default="true" doc="The number of reps" />
<arg name="hand" default="true" doc="The number of reps" />


<!-- Load controller settings -->

    <node name="pick_and_place_node" pkg="hand_arm" type="pick_place_run.py" respawn="false"
    	output="screen">
      	<param name="traj_profile" type="str" value="$(arg traj)" />
      	<param name="speed_factor" type="double" value="$(arg speed_factor)" />
      	<param name="num_reps" type="int" value="$(arg reps)" />
      	<param name="replan" type="bool" value="$(arg replan)" />
      	<param name="use_arm" type="bool" value="$(arg arm)" />
      	<param name="use_hand" type="bool" value="$(arg hand)" />
    </node>


</launch>